src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -139,6 +163,16 @@ internal struct CURLMsg
             internal int result;
         }
 
+        // Curl Version Information
+        [StructLayout(LayoutKind.Explicit)]
+        internal struct curl_version_info_data
+        {
+            [FieldOffset(0)]
+            internal int age;
+            [FieldOffset(32)]
+            internal int features;
+        }
```

Comment:
Yes. We are interested only in the `features` field. In the odd case, where the libcurl installed is very old, we do not want `Marshal.PtrToStructure` (the structure will not contain all the fields). That is why we are deliberately using `LayoutKind.Explicit`.